src/analytics/ledger/output_activity.rs

```diff
@@ -6,33 +6,30 @@ use std::collections::HashSet;
 use derive_more::{AddAssign, SubAssign};
 
 use super::TransactionAnalytics;
-use crate::types::{
-    ledger::{LedgerOutput, LedgerSpent, MilestoneIndexTimestamp},
-    stardust::block::{
-        output::{AliasId, NftId},
-        Address, Output,
+use crate::{
+    db::collections::analytics::{AliasActivityAnalyticsResult, NftActivityAnalyticsResult},
+    types::{
+        ledger::{LedgerOutput, LedgerSpent, MilestoneIndexTimestamp},
+        stardust::block::{
+            output::{AliasId, NftId},
+            Address, Output,
+        },
     },
 };
 
 /// Nft activity statistics.
 #[derive(Copy, Clone, Debug, Default, PartialEq, AddAssign, SubAssign)]
-pub struct NftActivityMeasurement {
-    pub created_count: u64,
-    pub transferred_count: u64,
-    pub destroyed_count: u64,
-}
-
-/// Measures the ledger Nft activity.
-#[derive(Debug)]
 pub struct NftActivityAnalytics {
-    measurement: NftActivityMeasurement,
+    created_count: u64,
```

Comment:
Similarly, we should be using `usize` here and in all other newly defined `*Analytics` structs.